libsolidity/experimental/codegen/IRGeneratorForStatements.h

```diff
@@ -34,6 +38,62 @@ class IRGeneratorForStatements: public ASTConstVisitor
 	IRGeneratorForStatements(IRGenerationContext& _context): m_context(_context) {}
 
 	std::string generate(ASTNode const& _node);
+
+	static std::size_t stackSize(IRGenerationContext const& _context, Type _type)
```

Comment:
This is large enough to warrant moving it to IRGeneratorForStatements.cpp - then you may also be able to avoid pulling in at least some of the three new headers at the top here :-).